test/representers/concerns/uri_methods_test.rb

```diff
@@ -5,10 +5,28 @@
 
 describe UriMethods do
 
-  let(:helper) { class TestUriMethods; include UriMethods; end.new }
+  let(:helper) { class TestUriMethods; include Rails.application.routes.url_helpers; include UriMethods; end.new }
   let(:t_object) { TestObject.new("test", true) }
   let(:representer) { Api::TestObjectRepresenter.new(t_object) }
 
+
+  it 'gets the path for url to represented' do
+    representer.prx_model_web_path(t_object).must_equal 'test_objects/1'
+  end
+
+  it 'helps an object become the represented base class' do
+    class FooParent; end
+    class FooChild < FooParent
+      def self.base_class; FooParent; end
+      def becomes(klass); 'became'; end
```

Comment:
Unused method argument - `klass`. If it's necessary, use `_` or `_klass` as an argument name to indicate that it won't be used. You can also write as `becomes(*)` if you want the method to accept any arguments but don't care about them.